sample-eksctl-ssh.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: bottlerocket
+  region: us-west-2
+  version: '1.15'
+
+nodeGroups:
+  - name: ng-bottlerocket
+    instanceType: m5.large
+    desiredCapacity: 4
+    amiFamily: Bottlerocket
+    ami: auto-ssm
+    iam:
+       attachPolicyARNs:
+          - arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy
+          - arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy
+          - arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly
+          - arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+    ssh:
+        allow: true
+        publicKeyName: YOUR_EC2_KEYPAIR_NAME
+    bottlerocket:
+      enableAdminContainer: true
```

Comment:
When `nodeGroup[0].ssh.allow` is `true`, this field isn't required to be set.

Reply:
Double-checking this now, thanks.